CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to React Native Sample APP
+
+Please follow this guidelines to start contributing to this project.
+
+# Creating a pull request
+
+1. Create a new branch from **main** branch following the name convention.
+2. Use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) convention for commit messages.
+3. Create a PR and include any useful context of the change in the description. Include any relevant screenshots.
+4. Wait for the PR to be reviewed and approved.
+
+# Branch naming convention
+
+The branch name should include the category, the Jira ticket number and a short description using "-" as separator.
+
+Use the following format: {category}/{Jira tickket number}-{short description}.
+
+Follow conventional commit categories for branch naming:
+
+| Category  | Meaning                                             |
+|-----------|-----------------------------------------------------|
+| fix       | for fixing a bug                                    |
+| feat      | for adding, removing or modifying a feature         |
+| test      | for experimenting something which is not an issue   |
+| wip       | for a work in progress                              |
```

Comment:
I'm not sure how often we will need to use the `wip` category, usually, we can add the "work in progress" label on a PR.

Reply:
make sense, I'll remove it